easyddb_event_status.module

```diff
@@ -52,15 +52,16 @@ function easyddb_event_status_preprocess(&$variables, $hook) {
   if (strpos($hook, 'ding_nodelist.ding_event.') !== FALSE) {
     $item = $variables['item'];
     if ($item->type == 'ding_event') {
-      $label = $item->field_ding_event_status_label[LANGUAGE_NONE][0]['value'];
+      if (isset($item->field_ding_event_status_label[LANGUAGE_NONE])) {
```

Comment:
Let's remove one level of nesting by combining the clauses:
`$item->type == 'ding_event' && isset($item->field_ding_event_status_label[LANGUAGE_NONE])`

Also, might consider drupal-way of doing things:
`$item->type == 'ding_event' && $label = field_get_items('node', $item, 'field_ding_event_status_label')[0]['value'] ?? FALSE`